nestkernel/streamers.h

```diff
@@ -0,0 +1,47 @@
+/*
+ *  streamers.h
```

Comment:
I don't like the name `streamers`.  How about `prettyprint`?

Reply:
The function does not print anything, so I would not call it "*print". The function is an operator with any ostream object as first argument. The term "stream" in this context is C++ convention. ...and just dumping an array to a stream is not "pretty" in my opinion. Would you rather like a different pre-/postfix than "vector[" and "]"?